settings/ChangePassword/ChangePassword.js

```diff
@@ -90,16 +90,30 @@ class ChangePassword extends Component {
     this.callout.sendCallout({ message: successMessage });
   };
 
-  handleChangePasswordError = err => {
-    const isWrongCurrentPassword = err.status === 401;
+  handleChangePasswordError = async (response) => {
```

Comment:
I do not think async is needed here. `response.json` is a synchronous operation.